.github/workflows/tiobe.yaml

```diff
@@ -0,0 +1,39 @@
+name: TIOBE Quality Checks
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 7 1 * *'
+
+jobs:
+  TICS:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+  
+      - name: Install dependencies
+        run: |
+          go install honnef.co/go/tools/cmd/staticcheck@v0.5.1
+          go get github.com/axw/gocov/gocov
+          go get github.com/AlekSi/gocov-xml
+
+      # We could store a report from the regular run, but this is cheap to do and keeps this isolated.
+      - name: Test and generate coverage report
+        run: |
+          go test -race -coverprofile=coverage.out ./...
```

Comment:
Do we need the race detector on here? It may not matter, but it makes the tests run somewhat slower, so I'd probably take it out here.